From f53817c1de61f5489e3dc42e13ae4526454f5c2d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maxime=20M=C3=A9r=C3=A9?= <maxime.mere@foss.st.com>
Date: Mon, 10 Jun 2024 14:02:20 +0200
Subject: [PATCH 633/841] Revert "feat(bl): align DATA section end on 2 bytes"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This reverts commit 04cf31a6264f01fbcf13ae53042eecb57d87aa79.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 7007e667691fef5e83d699b13ef0e78bd51c9712 Revert "feat(bl): align DATA section end on 2 bytes"]
Signed-off-by: Maxime Méré <maxime.mere@foss.st.com>
Change-Id: Ia8f59291027c32f194bad2425f9cf1cb78037c76
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/386386
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
---
 include/common/bl_common.ld.h | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/include/common/bl_common.ld.h b/include/common/bl_common.ld.h
index 6c1c69501..b6dd0f006 100644
--- a/include/common/bl_common.ld.h
+++ b/include/common/bl_common.ld.h
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020-2023, ARM Limited and Contributors. All rights reserved.
+ * Copyright (c) 2020-2022, ARM Limited and Contributors. All rights reserved.
  *
  * SPDX-License-Identifier: BSD-3-Clause
  */
@@ -121,7 +121,6 @@
 	.data . : ALIGN(DATA_ALIGN) {			\
 		__DATA_START__ = .;			\
 		*(SORT_BY_ALIGNMENT(.data*))		\
-		. = ALIGN(2);				\
 		__DATA_END__ = .;			\
 	}
 
-- 
2.47.2

